utils/build-script

```diff
@@ -541,20 +541,36 @@ class BuildScriptInvocation(object):
         # Currently we do not build external benchmarks by default.
         if args.build_external_benchmarks:
             impl_args += ["--skip-build-external-benchmarks=0"]
-        if not args.build_foundation:
-            impl_args += ["--skip-build-foundation"]
-        if not args.build_xctest:
-            impl_args += ["--skip-build-xctest"]
-        if not args.build_lldb:
-            impl_args += ["--skip-build-lldb"]
-        if not args.build_llbuild:
-            impl_args += ["--skip-build-llbuild"]
-        if not args.build_libcxx:
-            impl_args += ["--skip-build-libcxx"]
-        if not args.build_libdispatch:
-            impl_args += ["--skip-build-libdispatch"]
-        if not args.build_libicu:
-            impl_args += ["--skip-build-libicu"]
+
+        # If install_all is set, then products that we /always/ build (not by
+        # design) need to have their install flag set.
+        if args.install_all:
+            non_conditional_subprojects = [
```

Comment:
You can specify some combination of the following to avoid building any of these:

```
skip-build-swift
skip-build-llvm
skip-build-cmark
```

Indeed, we have a preset `buildbot_incremental,tools=RA,llvm-only` that only builds llvm. If it's not a lot of work, it would be good to handle that correctly.

Reply:
I think it got broken. Let me see.